apps/system/fota_delta_engine/Makefile

```diff
@@ -0,0 +1,144 @@
+###########################################################################
```

Comment:
Could you let me know why this is in apps?
In my opinion, it is not right that app updates binary in permission point of view.
And if this app binary needs to be updated, how does it work?